theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -120,12 +155,57 @@ let fundingAgenciesApp = new Vue({
       }
       return null;
     },
+    fetchFromRORList: async function (funderName) {
+      try {
+        let words = funderName.split(" ");
+        words = words.map((w) => encodeURIComponent(w));
+        this.filteredWords = [];
+        words = words.filter((word) => {
+          for (let char of this.removeCharsFromQuery) {
+            if (word.includes(char)) {
+              this.filteredWords.push(word);
+              return false;
+            }
+          }
+          return true;
+        });
+
+        let url = this.ROR_API_URL;
+        const params = new URLSearchParams({
+          query: words.join(" "),
+        })
+        if (this.LIMIT_US) {
+          params.append("filter", "country.country_code:US");
+        }
+        url = `${url}?${params.toString()}`;
+        const res = await fetch(url);
+        const result = await res.json();
+        result.items.forEach(item => {
+          item["alt-names"] = [];
+          item.names.forEach(nameItem => {
+            item["alt-names"].push(nameItem.value.toLowerCase());
+            if (nameItem.types.includes('ror_display')) {
+              item.name = nameItem.value;
+            }
+          });
+          item.url = item.id;
+        });
+        const funders = result.items;
+        this.crossreffApiDown = false;
+        return funders;
+      } catch (e) {
+        console.error(`Error querying Crossref API: ${e}`);
+        this.crossreffApiDown = true;
+      }
+      return null;
+    },
+
     initCrossrefFundersSearch: async function (funderName) {
```

Comment:
also rename this function, to make it clear that we are no longer searching the Crossref funder list